WordImpress/wp-business-reviews
# feat(builder): allow Place ID to be entered directly #194

## Description

Resolves #194 by allowing a Place ID to be entered directly into the Google search field.

New logic has been added that assumes the search term is a Place ID if the query has no spaces and is more than 24 characters long.

## Testing

1. Add a new Google collection.
2. Paste this Place ID into the search field and click Search: `ChIJjaEeqKeLUocRb9AmOfknIiM`
3. The business should immediately populate along with its reviews. The rest of the collection behavior is the same as any other collection.
